packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -334,39 +334,48 @@ Future<void> buildGradleApp({
   }
   command.add(assembleTask);
 
-  final Stopwatch sw = Stopwatch()..start();
-  int exitCode = 1;
   GradleHandledError detectedGradleError;
   String detectedGradleErrorLine;
+  final StringConverter consumeLog = (String line) {
```

Comment:
Nit: prefer a (mostly) consistent style of function declaration.

```suggestion
  String consumeLog(String line) {
```